view/next-project/src/utils/createSponsoractivitiesReceiptPDF.tsx

```diff
@@ -0,0 +1,264 @@
+                ご担当 : {sponsorActivitiesViewItem.sponsor.representative} 様
+              </Text>
+              <Text style={[styles.text_S, styles.underLine]}>
+                件名 : <Text style={styles.text_M}>技大祭企業協賛</Text>
+              </Text>
+              <View>
+                <Text style={[styles.text_S, styles.paddingTop]}>
+                  下記の通り、ご請求申し上げます。
+                </Text>
+                <View style={styles.sumField}>
+                  <Text style={styles.text_S}>
+                    合計金額 <Text style={styles.text_L}>¥ {totalPrice}</Text>
+                  </Text>
+                </View>
+              </View>
+            </View>
+            <View>
+              <Text style={styles.marginButtom}>
+                請求日 : {formatDate(sponsorActivitiesViewItem.sponsorActivity.createdAt || '')}
+              </Text>
+              <View>
+                <View style={styles.marginButtom}>
+                  <Text>技大祭実行委員</Text>
+                  <Text>〒940-2137</Text>
+                  <Text>新潟県長岡市上富岡町1603-1</Text>
+                  <Text>長岡技術科学大学 大学集会施設</Text>
+                </View>
+                <Text style={styles.text_S}>E-Mail : nutfes_shogai_kyosan@googlegroups.com</Text>
+                <Text>担当 : {sponsorActivitiesViewItem.user.name}</Text>
+              </View>
+            </View>
+          </View>
+        </View>
+        <View style={[styles.itemsTable, styles.marginButtom]}>
+          <View style={styles.tableRow}>
+            <Text style={[styles.commonTableCol, styles.tableColHeader_S]}>No.</Text>
+            <Text style={[styles.commonTableCol, styles.tableColHeader_L]}>概要</Text>
+            <Text style={[styles.commonTableCol, styles.tableColHeader_S]}>数量</Text>
+            <Text style={[styles.commonTableCol, styles.tableColHeader_M]}>単価</Text>
+            <Text style={[styles.commonTableCol, styles.tableColHeader_M]}>金額</Text>
+          </View>
+          {sponsorActivitiesViewItem.styleDetail.map((styleDetail, index) => (
+            <View style={styles.tableRow} key={index}>
+              <Text style={[styles.commonTableCol, styles.tableCol_S]}>{index + 1}</Text>
+              <Text style={[styles.commonTableCol, styles.tableCol_L]}>
+                {styleDetail.sponsorStyle.style} ({styleDetail.sponsorStyle.feature})
+              </Text>
+              <Text style={[styles.commonTableCol, styles.tableCol_S]}>1</Text>
+              <Text style={[styles.commonTableCol, styles.tableCol_M]}>
+                ¥ {styleDetail.sponsorStyle.price}
+              </Text>
+              <Text style={[styles.commonTableCol, styles.tableCol_M]}>
+                ¥ {styleDetail.sponsorStyle.price}
+              </Text>
+            </View>
+          ))}
+          <View style={styles.tableRow}>
+            <Text style={[styles.commonTableCol, styles.tableColHeader_S]}></Text>
+            <Text style={[styles.commonTableCol, styles.tableColHeader_L]}></Text>
```

Comment:
```suggestion
            <Text style={[styles.commonTableCol, styles.tableColHeader_S]} />
            <Text style={[styles.commonTableCol, styles.tableColHeader_L]} />
```
私はこっちのほうが好き
直すかはお任せします